service/src/main/java/bio/terra/pipelines/service/PipelinesService.java

```diff
@@ -41,14 +47,116 @@ public Pipeline getPipeline(PipelinesEnum pipelineName) {
    * This method is meant to only be called by an admin endpoint to update the control workspace id
    * for a pipeline
    *
-   * @param pipelineName - nanme of pipeline to update
+   * @param pipelineName - name of pipeline to update
    * @param workspaceId - UUID of workspace to update to
-   * @return
+   * @return pipeline with updated workspaceId
    */
   public Pipeline updatePipelineWorkspaceId(PipelinesEnum pipelineName, UUID workspaceId) {
     Pipeline pipeline = getPipeline(pipelineName);
     pipeline.setWorkspaceId(workspaceId);
     pipelinesRepository.save(pipeline);
     return pipeline;
   }
+
+  public void validateInputs(PipelinesEnum pipelineName, Object inputs) {
+    Pipeline pipeline = getPipeline(pipelineName);
+    List<PipelineInputDefinition> inputDefinitions = pipeline.getPipelineInputDefinitions();
+
+    LinkedHashMap<String, Object> inputsMap = castInputsToMap(inputs);
```

Comment:
Is this a `LinkedHashMap` to preserve order of the keys?  If not, it's generally better to just define it as map 
```suggestion
    Map<> inputsMap = castInputsToMap(inputs);
```

or even 
```suggestion
    var inputsMap = castInputsToMap(inputs);
```